src/components/views/TablesEventReservation/TablesEventReservation.js

```diff
@@ -1,17 +1,104 @@
 import React from 'react';
-import styles from './TablesEventReservation.scss';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import Button from '@material-ui/core/Button';
 
-const TablesEventReservation = () => (
-  <div className={styles.component}>
-    <h2>TablesEventReservation view</h2>
-    <h3>{getIdfromLink()}</h3>
-  </div>
-);
+import styles from './TablesEventReservation.module.scss';
 
-function getIdfromLink() {
-  var viewURL = window.location.href;
-  const id = viewURL.split('/').at(-1);
-  return id;
-}
+export default function TablesEventReservation() {
+  const [value, setValue] = React.useState('');
+
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
+
+  function getIdfromLink() {
```

Comment:
Ponieważ ten fragment kodu powtarza się bardzo często to najlepiej to wydzielić do sobnego pliku u zaimportować tę funkcję. Jesli bedziesz chciał zmienić coś w funkcji to masz jedno a nie np kilkanaście